ArrayOps.java

```diff
@@ -1,26 +1,173 @@
 public class ArrayOps {
     public static void main(String[] args) {
-        
+        //int[] array={1,-2,3,-4,5};
+        //System.out.println(findMissingInt(array));
+        //System.out.println(secondMaxValue(array));
+        int [] array2={1,2,3,-4,5};
+        int [] array1={1,3,-4,5};
+        System.out.println(containsTheSameElements(array1,array2));
+        //int [] array={1,1,1,2,2};
+        //System.out.println(isSorted(array));
     }
     
+    //returns the missing integer in the array, between 0 and the array length
     public static int findMissingInt (int [] array) {
         // Write your code here:
-        return -1;
+        int sumArray=0;
+        int supposedSumArray=0;
```

Comment:
If you done this, you could avoid 2 loops - check the next comment
```suggestion
        int supposedSumArray=array.length;
```